core/spec.md

```diff
@@ -577,67 +588,45 @@ Then the existing entity can be deleted.
 - Examples:
   - `https://example2.com/myregistry/endpoints/9876`
 
-#### `createdby`
-
-- Type: String
-- Description: A reference to the user or component that was responsible for
-  the creation of this entity. This specification makes no requirement on
-  the semantics or syntax of this value
-- Constraints:
-  - If present, MUST be non-empty
-  - MUST be a read-only attribute in API view
-  - If this feature is enabled after an entity has already been created,
-    that entity's `createdby` value MUST remain empty
-- Examples:
-  - `John Smith`
-  - `john.smith@example.com`
-
 #### `createdat`
 
 - Type: Timestamp
 - Description: The date/time of when the entity was created
 - Constraints:
   - MUST be a [RFC3339](https://tools.ietf.org/html/rfc3339) timestamp
-  - MUST be a read-only attribute in API view
-  - If this feature is enabled after an entity has already been created,
-    that entity's `createdat` value MUST remain empty
+  - When present in a write operation request, the value MUST override any
+    existing value, however a value of `null` MUST use the current date/time
+    as the new value
+  - When absent in a write operation request, any existing value MUST remain
+    unchanged, or if not present, set to the current date/time
+  - In cases where multiple entity's `createdat` attribute is set to the
+    current date/time within the same operation, the same value MUST be
+    applied to all of the entities
```

Comment:
```suggestion
  - In cases where the `createdat` attribute is set to the
    current date/time on multiple entities within the same operation, the same value MUST be
    applied to all of the entities
```

Reply:
(Current "multiple entity's" doesn't work in terms of plural/singular; "multiple entities'" might be okay but looks clumsy to me.)